src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2571,7 +2571,15 @@ namespace winrt::TerminalApp::implementation
         // If we're holding the settings tab's switch command, don't create a new one, switch to the existing one.
         if (!_switchToSettingsCommand)
         {
-            auto newTabImpl = winrt::make_self<SettingsTab>(_settings);
+            winrt::Microsoft::Terminal::Settings::Editor::MainPage sui{ _settings };
+            sui.OpenJson([weakThis{ get_weak() }](auto&& /*s*/, auto&& /*e*/) {
+                if (auto page{ weakThis.get() })
+                {
+                    page->_LaunchSettings(SettingsTarget::SettingsFile);
```

Comment:
How should we let users open defaults.json?

Reply:
We could do the same Alt trick